packages/flutter_splash_tizen/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.example.flutter_splash_tizen_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="wearable"/>
```

Comment:
Does this package support TV as well?

Reply:
Could you confirm whether splash screen is supported on TV devices or not? If not, you should mention the fact in the package's README.

Reply:
@swift-kim As far as I see - it does not run on TV. It runs on mobile and on Wearable fine.

Reply:
Will this work for that?
> Package has been tested on Mobile 6.0, Wearable 6.0 and IoT Headed 6.5 platform.